### ISSUE TITLE
Titus: python setup.py test failure

### ISSUE THREAD

**Issue by sigstop** [opendatagroup/hadrian#1]
The following failed on MacOSX El Capitan:

```
$ cd hadrian/titus
$ python setup.py test
```
...

```
======================================================================
FAIL: testUnpackBoolean (test.testGenpy.TestGeneratePython)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/me/Git/hadrian/titus/test/testGenpy.py", line 2108, in testUnpackBoolean
    self.assertEqual(engine.action("".join(map(unsigned, [8]))), True)
AssertionError: False != True

----------------------------------------------------------------------
Ran 805 tests in 40.097s

FAILED (failures=1, errors=17)
$
```

**Comment by collinbennett** [comment]
This is a bug introduced with El Capitan 10.11 and how libraries under /usr are loaded.  The fix is in
commit: fe5cb465592455f0cb230e8b592ef587bb9395bb

I've tested it on a machine running 10.10, a machine upgraded to 10.11, and a new machine that shipped with 10.11.  This does not impact running under linux.

When running the tests on a Mac, there are 5 failures all in the regex package.  This has to do with differences between the underlying libraries in Darwin and Linux.  I will open issues for these so that people know they are there.  All other tests run successfully on Mac and Linux.

**Comment by sigstop** [comment]
Thanks! I just test tested on ```fe5cb46``` on OSX El Capitan 10.11.1 (upgraded from Yosemite) and it worked and reported the 5 failures that you mentioned:

```
$ uname -a
Darwin mybox.myco.com 15.0.0 Darwin Kernel Version 15.0.0: Sat Sep 19 15:53:46 PDT 2015; root:xnu-3247.10.11~1/RELEASE_X86_64 x86_64
...
$ python setup.py test
...
Ran 805 tests in 40.468s

FAILED (failures=5)
$
